project/Dependencies.scala

```diff
@@ -93,6 +97,7 @@ object Dependencies {
   val codegen = l ++= Seq(
     Compile.scalapbCompilerPlugin,
     Protobuf.protobufJava, // or else scalapb pulls older version in transitively
+    Protobuf.googleApi, // or else scalapb pulls older version in transitively
```

Comment:
Why does the codegen need a dependency on the common protos?

If we should always transitively pull it in, that should be from the (Java?) runtime rather, but should we have it as a transitive out of the box at all?

Reply:
You are right, not needed.